### PR TITLE
DET-1421: Change TEST_DISALLOWED to OS_PREVENTED_EXECUTION

### PR DESCRIPTION
**Pull request by keitharobertson**

### CODE REVIEW

**Review by makk94 — APPROVED** [review]

**Comment by stephanwampouille** [comment]
kk

File: python/sdk/prelude_sdk/models/codes.py [hunk 1]
```diff
@@ -128,7 +128,7 @@ def mapping(cls):
                 ExitCode.PROCESS_BLOCKED_GRACEFULLY,
                 ExitCode.PROTECTED,
                 ExitCode.STATIC_QUARANTINE,
-                ExitCode.TEST_DISALLOWED,
+                ExitCode.OS_PREVENTED_EXECUTION,
```
Comment (makk94) [inline python/sdk/prelude_sdk/models/codes.py:131]: alphabetize?

Comment (keitharobertson) [reply python/sdk/prelude_sdk/models/codes.py:131]: One day I'll learn my ABCs I promise

**Comment by stephanwampouille** [comment]
FE is good to merge this whenever

**Pull request merged**